### PR TITLE
Goodbye free delimb. 

### PR DESCRIPTION
**Pull request by InterroLouis**
## About The Pull Request

Warrior delimb gone. 

## Why It's Good For The Game

Because it is. 

## Changelog
:cl:
del: Warrior delimb gone
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -2. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Kuro-ai** [comment]
only ever useful for meta targetting good marines,it doesn't actually add anything to warrior,this can go imo

**Comment by David-Stormwell** [comment]
Time to camp captures harder

File: code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm [hunk 1]
```diff
@@ -284,74 +284,3 @@
...
-// *********** Rip limb
-// ***************************************
-
-// Called when pulling something and attacking yourself with the pull
-/mob/living/carbon/xenomorph/proc/pull_power(mob/M)
-	if (isxenowarrior(src) && !ripping_limb && M.stat != DEAD)
-		ripping_limb = TRUE
-		if(rip_limb(M))
-			stop_pulling()
-		ripping_limb = FALSE
```
Comment (psykzz) [inline code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm:298]: do you not need to remove the `ripping_limb ` define too, or is it used for something else?

**Pull request merged**